client/jetpack-connect/main.jsx

```diff
@@ -243,11 +244,25 @@ export class JetpackConnectMain extends Component {
 		);
 	}
 
+	isError( error ) {
+		return (
+			this.state.currentUrl &&
+			this.isCurrentUrlFetched() &&
+			this.props.jetpackConnectSite &&
+			this.props.jetpackConnectSite.error &&
+			this.props.jetpackConnectSite.error.error === error
```

Comment:
Could use `_.get()` for the last three lines.

Reply:
Good call 👍 done in https://github.com/Automattic/wp-calypso/pull/29983/commits/239806677baadbaae57deacb34638fbe75fb8e15.